openedx_ledger/admin.py

```diff
@@ -105,6 +105,34 @@ def get_quantity_usd(self, obj):
         return None
 
 
+class TransactionTypeFilter(admin.SimpleListFilter):
```

Comment:
This custom list filter a bonus thing that helped me debug locally, and might be useful to us in general.
![Screenshot 2024-07-01 at 14 14 50](https://github.com/openedx/openedx-ledger/assets/85151/8c640c27-56e7-48f7-8bcb-3f776aaa95a1)